emulators/hyperkit/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                hyperkit
+version             0.20210107
+categories          emulators
+platforms           darwin
+maintainers         nomaintainer
+license             BSD
+
+description         HyperKit is a toolkit for embedding hypervisor capabilities in your application.
+
+long_description    HyperKit is a toolkit for embedding hypervisor capabilities in your application. \
+                    It includes a complete hypervisor, based on xhyve/bhyve, which is optimized for \
+                    lightweight virtual machines and container deployment. It is designed to be interfaced \
+                    with higher-level components such as the VPNKit and DataKit.
+
+
+
```

Comment:
we probably don't need three newlines here ;)

Reply:
😬